sklearn/utils/testing.py

```diff
@@ -94,8 +95,7 @@
 _dummy = TestCase('__init__')
 assert_equal = _dummy.assertEqual
 assert_not_equal = _dummy.assertNotEqual
-assert_true = _dummy.assertTrue
-assert_false = _dummy.assertFalse
+
```

Comment:
please avoid irrelevant changes

Reply:
I removed the extra spaces and edited the message. I'll look into how to add a test. Thanks for your feedback!

Reply:
Maybe something along these lines? I'm still not too familiar with the code base.

`
def test_assert_true():
    assert_warns(DeprecationWarning, assert_true, 1)


def test_assert_false():
    assert_warns(DeprecationWarning, assert_false, 0)
`